x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/ComputeService.java

```diff
@@ -431,7 +432,7 @@ Runnable cancelQueryOnFailure(CancellableTask task) {
         });
     }
 
-    CancellableTask createGroupTask(Task parentTask, Supplier<String> description) {
+    CancellableTask createGroupTask(Task parentTask, Supplier<String> description) throws TaskCancelledException {
```

Comment:
Isn't `TaskCancelledException` a runtime exception? So making it checked here doesn't do much except documenting the fact it may be thrown, am I missing something?

Reply:
Yes, that's correct.